aws/resource_aws_kinesis_firehose_delivery_stream_test.go

```diff
@@ -862,6 +892,34 @@ func TestAccAWSKinesisFirehoseDeliveryStream_ExtendedS3Updates(t *testing.T) {
 	})
 }
 
+func TestAccAWSKinesisFirehoseDeliveryStream_ExtendedS3_KinesisStreamSource(t *testing.T) {
```

Comment:
Need to add a separate acceptance test rather than just adding an import check to `TestAccAWSKinesisFirehoseDeliveryStream_s3KinesisStreamSource` as import is not support for `destination = "s3"`.